### PR TITLE
Highlight improvements

### PR DESCRIPTION
**Pull request by akintolga**
[SD-4164] - Display highlight preview
[SD-4170] - Process first paragraphs of highlight stories
[SD-4086] - Privilege check for displaying "remove highlight button"
[SD-3826] - Highlight set up is simplified to use "main" group only

### CODE REVIEW

**Comment by akintolga** [comment]
Depends on https://github.com/superdesk/superdesk-client-core/pull/140

**Comment by mdhaman** [comment]
:+1:

File: scripts/superdesk-authoring/authoring.js [hunk 1]
```diff
@@ -1065,6 +1071,19 @@
                         });
                 }
 
+                function _previewHighlight(_id) {
+                    api.generate_highlights.save({}, {'package': _id, 'preview': true})
+                    .then(function(response) {
+                        $scope.highlight_preview = $sce.trustAsHtml(response.body_html);
```
Comment (ahilles107) [inline scripts/superdesk-authoring/authoring.js]: @vied12 we don't have special way of dealing with trusted html? I remember that i saw you asking about that somewhere?

File: scripts/superdesk-packaging/packaging.less [hunk 2]
```diff
@@ -214,6 +214,9 @@
                 padding: 40px 0;
             }
         }
+        .preview-container {
+        	padding: 20px 20px;
```
Comment (ahilles107) [inline scripts/superdesk-packaging/packaging.less]: this file have mixed tab size (sometimes it's 4 spaces, sometimes 4 tabs). Can you fix it? Even in this line it's 8 spaces and 1 tab.

File: scripts/superdesk-authoring/authoring.js (hunk 1 above)
Comment (vied12) [inline scripts/superdesk-authoring/authoring.js]: We have the `sdHtmlPreview` directive for that

**Comment by petrjasek** [comment]
:+1:

**Pull request merged**